src/languages/lsl.js

```diff
@@ -0,0 +1,133 @@
+                end: '[{;]',
+                excludeEnd: true,
+                contains: [
+                    {
+                        className: 'params',
+                        begin: '\\(',
+                        end: '\\)',
+                        contains: [
+                          'self',
+                          'constant.float',
+                          'constant.integer',
+                          'constant.quaternion',
+                          'constant.string',
+                          'constant.vector',
+                          'string',
+                          'number'
+                        ]
```

Comment:
`contains` should contains actual JavaScript objects, not names (with the exception of 'self', which is a syntactic sugar). To reuse numbers, strings, etc. assign them to regular variables beforehand. Look into, say, https://github.com/isagalaev/highlight.js/blob/master/src/languages/cpp.js for an example.